src/Header/Header.tsx

```diff
@@ -1,29 +1,53 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './Header.css';
-import logo from '../assets/logo.png';
-import { Navbar, Nav, Button } from 'react-bootstrap';
-import { useHistory } from 'react-router-dom';
+import folderIcon from '../assets/headerIcons/folderIcon.svg';
 
 const Header = () => {
-  const history = useHistory();
+  const [Menu, setMenu] = useState(false);
+
+  function openMenu() {
```

Comment:
Maybe rename this `toggleMenu`? Since it's not just opening it, it's also possible that is closes the menu.